app/_src/kubernetes-ingress-controller/guides/configure-acl-plugin.md

```diff
@@ -3,72 +3,63 @@ title: Configuring ACL Plugin
 content_type: tutorial
 ---
 
-This guide walks through configuring the Kong ACL Plugin. The ACL Plugin
-requires the use of at least one Authentication plugin. This example uses
-the JWT Auth Plugin.
+Learn to configure the Kong ACL Plugin. To use the ACL Plugin you need at least one Authentication plugin. This example uses the JWT Auth Plugin.
 
-{% include_cached /md/kic/installation.md kong_version=page.kong_version %}
-
-{% include_cached /md/kic/http-test-service.md kong_version=page.kong_version %}
+{% include_cached /md/kic/prerequisites.md kong_version=page.kong_version disable_gateway_api=false%}
 
 {% include_cached /md/kic/class.md kong_version=page.kong_version %}
```

Comment:
Do we need the `Create a configuration group` section? The `helm` install creates this resource automatically

Reply:
removed.